test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -37,7 +37,7 @@ public EndToEndTests(ServerFixture serverFixture)
 
         [ConditionalFact]
         [OSSkipCondition(OperatingSystems.Windows, WindowsVersions.Win7, WindowsVersions.Win2008R2, SkipReason = "No WebSockets Client for this platform")]
-        public async Task WebSocketsTest()
+        public async Task BareWebSocketsTest()
```

Comment:
Just to be a little more explicit that this test is using bare websockets and not testing the transports. I can change this back